src/utils.ts

```diff
@@ -5,3 +5,15 @@ export const formatColour = (colour: string) => {
     return colour;
   }
 };
+
+export const processRegExpPattern = (pattern: string) => {
+  const matchDelimiters = pattern.match(/^\/(.*)\/(.*)$/);
+
+  const regexp = (matchDelimiters || []).slice(1);
+  const flags = regexp.pop();
+  const source = regexp.join('');
+
+  return flags
+    ? RegExp.apply(RegExp, [source, flags])
+    : pattern;
```

Comment:
I think this function should always return a RegExp, so the condition handlers can just do `const pattern = processRegExpPattern(condition.pattern);`

```suggestion
    : new RegExp(pattern);
```

Reply:
You're right, thats make more sense and its more clear solution. Fixed !